### ISSUE TITLE
Pin Jinja2 to fix docs build

### ISSUE THREAD

**Issue by hugovk** [python/cpython#91294]
BPO | [47138](https://bugs.python.org/issue47138)
--- | :---
Nosy | @ned-deily, @ambv, @zooba, @hugovk, @miss-islington, @m-aciek
PRs | <li>python/cpython#32111</li><li>python/cpython#32109</li><li>python/cpython#32153</li><li>python/cpython#32154</li><li>python/cpython#32161</li><li>python/cpython#32163</li><li>python/cpython#32167</li><li>python/cpython#32168</li><li>python/cpython#32169</li><li>python/cpython#32182</li><li>python/cpython#32183</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2022-03-28.08:17:25.467>
labels = ['3.7', '3.8', '3.9', '3.10', 'docs']
title = 'Pin Jinja2 to fix docs build'
updated_at = <Date 2022-04-08.13:06:24.310>
user = 'https://github.com/hugovk'
```

bugs.python.org fields:
```python
activity = <Date 2022-04-08.13:06:24.310>
actor = 'lukasz.langa'
assignee = 'docs@python'
closed = False
closed_date = None
closer = None
components = ['Documentation']
creation = <Date 2022-03-28.08:17:25.467>
creator = 'hugovk'
dependencies = []
files = []
hgrepos = []
issue_num = 47138
keywords = ['patch']
message_count = 14.0
messages = ['416149', '416151', '416158', '416167', '416171', '416194', '416195', '416196', '416197', '416209', '416233', '416235', '416285', '416983']
nosy_count = 7.0
nosy_names = ['ned.deily', 'docs@python', 'lukasz.langa', 'steve.dower', 'hugovk', 'miss-islington', 'Maciej Olko']
pr_nums = ['32111', '32109', '32153', '32154', '32161', '32163', '32167', '32168', '32169', '32182', '32183']
priority = 'normal'
resolution = 'fixed'
stage = 'patch review'
status = 'open'
superseder = None
type = None
url = 'https://bugs.python.org/issue47138'
versions = ['Python 3.7', 'Python 3.8', 'Python 3.9', 'Python 3.10']
```

</p></details>

**Comment by kumaraditya303** [comment]
@hugovk Can this be closed now ?

**Comment by hugovk** [comment]
We're still waiting for https://github.com/python/cpython/pull/32183 to be merged for 3.8.

Ping 3.8 RM @ambv.
